gtfs_digest/_gtfs_digest_specific_datasets.py

```diff
@@ -45,7 +45,13 @@ def get_day_type(date):
     """
     Function to return the day type (e.g., Monday, Tuesday, etc.) from a datetime object.
     """
-    days_of_week = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]
+    days_of_week = ["Monday", 
```

Comment:
Repeats `segment_speed_utils.time_helpers`, days of week are already set up in a dictionary. example: https://github.com/cal-itp/data-analyses/blob/main/rt_segment_speeds/segment_speed_utils/gtfs_schedule_wrangling.py#L114